pkg/apis/kops/cluster.go

```diff
@@ -75,6 +75,9 @@ type ClusterSpec struct {
 	// NetworkID is an identifier of a network, if we want to reuse/share an existing network (e.g. an AWS VPC)
 	NetworkID string `json:"networkID,omitempty"`
 
+	// Set to true and a gateway will not be created.  In aws it is represented by an IGW.
+	NetworkSkipCreateGateway bool `json:"networkSkipCreateGateway,omitempty"`
```

Comment:
I think it's cleaner and more future proof to call this `NetworkCreateGateway` and default it to true